### DISCUSSION TITLE
How to bypass (Heur)Protection: Obfuscation[Modified EP + Anti-ILDASM + Math mutations + Watermark]

### DISCUSSION THREAD

**Discussion by testeralia** [x64dbg/x64dbg#3477] [Q&A]
hi, I started to learning reverse engineering so I need your help about the protection method :(Heur)Protection: Obfuscation[Modified EP + Anti-ILDASM + Math mutations + Watermark]
![bapass](https://github.com/user-attachments/assets/8b54c858-429a-4fb4-99a8-a88792103e59)

**Comment by Master-lev** [comment]
Я решил эту проблему с помощью программы de4dot-cex. Нужно перетащить защищенный файл (если их несколько в программе - то каждый) на de4dot-x64.exe. В папке программы, которую нужно вылечить, появится второй такой-же файл, но уже без защиты.
